Libraries/LibWeb/HTML/HTMLElement.cpp

```diff
@@ -925,6 +927,241 @@ void HTMLElement::adjust_computed_style(CSS::StyleProperties& style)
     }
 }
 
+// https://html.spec.whatwg.org/multipage/popover.html#check-popover-validity
+WebIDL::ExceptionOr<bool> HTMLElement::check_popover_validity(bool expected_to_be_showing, bool throw_exception, GC::Ptr<DOM::Document> expected_document)
```

Comment:
Rather than using `bool` arguments, we tend to use enums with `Yes` and `No` values.

Also, the `throw_exception` parameter is named `throw_exceptions` in the header file, which follows the name used in the spec.